docusaurus-search-local/src/client/theme/SearchPage/SearchPage.tsx

```diff
@@ -222,11 +222,12 @@ function SearchResultItem({
   if (!isTitle) {
     pathItems.push((page as SearchDocument).t);
   }
+  const hightLightWords = tokens.map(x=>"_highlight="+x).join("&");
```

Comment:
Hi, thanks for your work! But here we should escape the params, and we'd better decide whether to append the params according to `options.highlightSearchTermsOnTargetPage`.

Reply:
I just updated it